src/analysis/retail/hunter/beastmastery/modules/core/SpellUsable.tsx

```diff
@@ -34,6 +66,17 @@ class SpellUsable extends CoreSpellUsable {
         );
       }
     }
+    if (spellId === TALENTS.KILL_COMMAND_SHARED_TALENT.id && this._has4pc) {
+      if (this.isOnCooldown(spellId) && this.chargesAvailable(spellId) === 0) {
+        this.endCooldown(
+          spellId,
+          this.lastPotentialTriggerForKillCommandReset
+            ? this.lastPotentialTriggerForKillCommandReset.timestamp
+            : undefined,
```

Comment:
this can be simplified to:
```suggestion
          this.lastPotentialTriggerForKillCommandReset?.timestamp,
```